### DISCUSSION TITLE
2008/11/21/%E5%9C%A8%E7%BE%8E%E5%9B%BD%E8%B4%AD%E7%89%A9

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#249] [Announcements]
# 2008/11/21/%E5%9C%A8%E7%BE%8E%E5%9B%BD%E8%B4%AD%E7%89%A9

营养药和美容护肤品

https://blog.qizhen.xyz/2008/11/21/%E5%9C%A8%E7%BE%8E%E5%9B%BD%E8%B4%AD%E7%89%A9

**Comment by ruanqizhen** [comment]
之前博客下的留言：

ZPMANR
2009年07月12日 15:17 编辑
请问各位契尔氏Keil的东西在什么牌子的折扣店购买划算。我在佛吉尼亚的Herndon市。能邮件告诉我不？我的Email ：zpmanr@msn.com谢谢

回复
Hazel
2009年02月4日 17:49 编辑
其实有很多美国品牌的护肤品在国内航站楼的免税店买反而更便宜呢，遇到搞促销的时候还送一大堆。哈哈

回复
文婷
2009年01月1日 17:03 编辑
笑死我了～～～

回复
Chrise
2008年12月22日 10:02 编辑
很深刻！

回复
hongfei
2008年11月23日 10:37 编辑
最后一句话经典！

回复
yoyo
2008年11月22日 19:35 编辑
中国啥都贵 没办法啊 菜场里面的西红柿 黄瓜也不知道被施过什么好东东 用着胆战心惊

回复
Nicholas
2008年11月22日 14:36 编辑
跑去买便宜货了 啊～
 
其实倩碧在美国都属于奢侈品，olay和欧莱雅才是美国的大宝，沃尔玛有售。
最近我还抗了一堆东西回来，很麻烦，空着一个箱子去，回来得整两个大箱子才装的下。
 
我去的小城市，那里mall的人对中国人特欢迎，唉～看来中国人真是有钱了，不光是这些东西国内比人家贵，就连油价都是人家3倍了

回复
li
2008年11月22日 13:47 编辑
哈哈,美国现在买东西是很合算的呀
COACH的包包美国买也很超值,OUTLET还有对折的COACH卖呢,中国不知道搞什么,什么东西都卖很贵
黄瓜水、西红柿皮、西红柿瓤什么的…..应该是Kiels的小黄瓜水,番茄红素霜吧,都是很好用的产品

回复
雪狼
2008年11月22日 08:44 编辑
美国通了~

回复

Chrise
2008年12月22日 09:57 编辑
hehe, 同感同感！顺便去美国宣传一下LabVIEW Tips Video哦

回复
hongfei
2008年11月25日 11:47 编辑
看来你是要连载了啊，(*^__^*) 嘻嘻……
和你感触差不多，有了老婆后，便不再去超市买衣服了。不过东西确实不一样啊。还是有老婆好。

回复
秋
2008年11月24日 15:55 编辑
频频购购购~~

回复
lingyuan
2008年11月24日 09:58 编辑
最求品牌对于我们这种穷人来说简直是奢望，地摊货还是比较合适俺！

回复
雪狼
2008年11月23日 18:17 编辑
应聘去你公司

回复
Elaine
2008年11月23日 16:35 编辑
在美国购物，看的人心痒痒的。我同事上次去美国，买了8双鞋回来，其中一个就是clarks的。我也最近迷上了手表，可惜中国的太贵。

回复
li
2008年11月23日 15:06 编辑
你们公司还招人不?
中国就是卖的死贵~~~
从单身到有女朋友到老婆 衣服的历史完全改变了~~~这段比较经典
女人都是购物狂,帮自己买帮家人买都开心的…
SO女人的钱好赚啊

回复
Vivian
2008年11月23日 13:45 编辑
你在什么公司阿??

回复

心花怒放
2009年03月20日 13:28 编辑
呵呵，我也想买，可以接受５K-６K的。老师您的经验值得学习啊。

回复
Jucao
2008年11月27日 10:33 编辑
兄弟，怎么还买联想的东西？呵呵。自从以前用了联想的PC机深受其害之后，我想我应该不会再买任何联想的东西了。以前公司用联想电脑，据说故障率超过30%，而且多是硬盘损坏。联想如果不是政府扶持，应该早就倒掉了。现在的笔记本电脑，差异化越来越小了。远远看去，除了商标logo不同以外，都大同小异。Thinkpad以前是不错，现在到了联想手里，也被糟蹋得差不多了。苹果电脑的确是不错，既可以装Mac OS也可以装Windows，如果赶上购物季节打折，买老款的还是不算太贵了。（最便宜一款好像是800多美刀）

回复
miaoxin
2008年11月25日 17:18 编辑
富士通的性价比也很高，就是有点儿热。。。
好多服装品牌也都是，在国外明明是面向中低档消费人群，到中国就变成高档了。
看来在中国钱很好赚，是块肥肉啊！

回复
hongfei
2008年11月25日 11:43 编辑
联想高层已经说过：高端的就是要挣中国人的钱。
中国人的生活成本高是全世界有目共睹的。

回复
秋
2008年11月24日 15:55 编辑
购的爽啊~

回复
逸美
2008年11月24日 11:01 编辑
我用的电脑也是联想的.使用下来很好.

回复
lingyuan
2008年11月24日 09:52 编辑
呵呵，深有同感啊！在网上折扣买东西我总觉的没有安全感！因为要保修或者是出现故障的话还真是另人头疼的事情。好像现在在国外买电脑在价位上都会比在我们内地买便宜上很多，上次我同事去香港也从那边买了一部联想的本本，价位和我们内地的竟然差了很多，嘿嘿，这个能说明什么呢？ 差距真的是好大啊！

回复
